include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -92,12 +92,46 @@ class alignment_executor_two_way
      * \brief The class is not copy-constructible or copy-assignable but allows move construction and assignment.
      * \{
      */
-    alignment_executor_two_way() = delete;                                               //!< This is a move-only type.
-    alignment_executor_two_way(alignment_executor_two_way const &) = delete;             //!< This is a move-only type.
-    alignment_executor_two_way(alignment_executor_two_way &&) = default;                 //!< Defaulted
-    alignment_executor_two_way & operator=(alignment_executor_two_way const &) = delete; //!< This is a move-only type.
-    alignment_executor_two_way & operator=(alignment_executor_two_way && ) = default;    //!< Defaulted
-    ~alignment_executor_two_way() = default;                                             //!< Defaulted
+    //!\brief Explicitly deleted default constructor.
```

Comment:
Because it already states in the brief of the constructor that it is move-only. So for me it did not add more value to say it is move only type as it can easily happen to misread that it is deleted/defaulted in the documentation. So I wanted to express in the brief that the constructor is deleted on purpose. Does that make sense to you? Or should I change it back?

Reply:
as discussed